scripts/hail_batch/main.py

```diff
@@ -0,0 +1,28 @@
+"""Run HGDP_1KG_PCALoadingsGenerate.py using the analysis runner."""
```

Comment:
This is just a minor comment, but generally we use "snake-case" for filenames, e.g.
`hgdp_1kg_pca_loadings_generate.py` or maybe create a subdirectory for this use case, like
`hgdp_1kg/generate_pca_loadings.py`?

Reply:
that's really helpful, thanks! I've changed it to hgdp_1kg_pca_loadings_generate.py. I think next time I might try and think about structuring the folders a bit differently as well.